uniref-rest/src/test/java/org/uniprot/api/uniref/controller/TestUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.uniprot.api.uniref.controller;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream;
+
+/** Contains utility methods that aid in testing */
+public final class TestUtils {
+
```

Comment:
Use UniProtKB's TestUtils by moving it to common.

Reply:
I have created TestUtils class in common with uncompress method, as this is the only common method used by both uniref and uniprotkb. Rest of the UniProtKb TestUtils methods are being used by uniprotkb only, so keeping it there.